sw.js

```diff
@@ -1,5 +1,9 @@
 /* global workbox:false */
 
-self.importScripts('assets/js/vendor/{fileName}')
+self.importScripts('assets/js/vendor/workbox/{workboxVersion}/workbox-sw.js')
+
+workbox.setConfig({
+  modulePathPrefix: '/assets/js/vendor/workbox/{workboxVersion}/'
```

Comment:
This will fail with our current live docs structure. Can you make it relative?

Reply:
It is relative to the file, no?

Reply:
nope, it's relative to the server root :)